<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FV_ThresholdDlg</class>
 <widget class="QDialog" name="FV_ThresholdDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>400</width>
     <height>480</height>
    </rect>
   </property>
   <property name="title">
    <string>单元电流阀值</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>401</width>
     <height>480</height>
    </rect>
   </property>
   <property name="title">
    <string>单元电流阀值</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>510</y>
     <width>140</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>510</y>
     <width>140</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setAllBtn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>510</y>
     <width>140</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>统一设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="refreshBtn">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>510</y>
     <width>140</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
